### ISSUE TITLE
Early Errors ... build hangs on ES startup

### ISSUE THREAD

**Issue by fpurcell** [pelias/dockerfiles#31]
Not a lot has changed in the dockerfiles project since Sept 29 (last week) .. see comments at the bottom of #25.  But now I'm seeing some early errors, followed by the build hanging when trying to start Elastic Search.    See [hung-es.txt](https://github.com/pelias/dockerfiles/files/1357911/hung-es.txt)

Again, this is from a clean Docker system with all cached data, images, volumes, etc... nuked.

**Comment by fpurcell** [comment]
Note: switching from  'docker-compose build;' to 'docker-compose pull;' does successfully build a running instance of Pelias.  And unlike last week, Interpolation seems to be working.  So that's GREAT NEWS!!!  (just closed ticket #25).

This point http://cs-dv-mapgeo01:4000/v1/search?text=888%20SE%20Lambert%20St in last week's version was resulting only the two "match_type": "fallback" records (see below).  But today there's the expected "match_type": "interpolated" result in the results.  

One question: why does the result weight the interpolated and fallback results with the same 80% confidence score?  Would hope that interpolation result would be 80% and fallbacks below 50% (so that they could be thrown out, and the single interpolated result win out -- thinking out loud, I guess an app could just throw out fallbacks when other results are available).

http://cs-dv-mapgeo01:4000/v1/search?text=888%20SE%20Lambert%20St 
```json
{
"geocoding": {
"version": "0.2",
"attribution": "http://pelias.mapzen.com/v1/attribution",
"query": {
"text": "888 SE Lambert St",
"size": 10,
"private": false,
"lang": {
"name": "English",
"iso6391": "en",
"iso6393": "eng",
"defaulted": false
},
"querySize": 20,
"parsed_text": {
"number": "888",
"street": "se lambert st"
}
},
"engine": {
"name": "Pelias",
"author": "Mapzen",
"version": "1.0"
},
"timestamp": 1507175080699
},
"type": "FeatureCollection",
"features": [
{
"type": "Feature",
"geometry": {
"type": "Point",
"coordinates": [
-122.656975,
45.468603
]
},
"properties": {
"id": "polyline:20949",
"gid": "mixed:address:polyline:20949",
"layer": "address",
"source": "mixed",
"name": "888 Southeast Lambert Street",
"housenumber": "888",
"street": "Southeast Lambert Street",
"confidence": 0.8,
"match_type": "interpolated",
"accuracy": "point",
"country": "United States",
"country_gid": "whosonfirst:country:85633793",
"country_a": "USA",
"region": "Oregon",
"region_gid": "whosonfirst:region:85688513",
"region_a": "OR",
"county": "Multnomah County",
"county_gid": "whosonfirst:county:102081631",
"locality": "Portland",
"locality_gid": "whosonfirst:locality:101715829",
"neighbourhood": "Sellwood-Moreland",
"neighbourhood_gid": "whosonfirst:neighbourhood:85867151",
"label": "888 Southeast Lambert Street, Portland, OR, USA"
}
},
{
"type": "Feature",
"geometry": {
"type": "Point",
"coordinates": [
-122.604483,
45.467254
]
},
"properties": {
"id": "polyline:20948",
"gid": "openstreetmap:street:polyline:20948",
"layer": "street",
"source": "openstreetmap",
"source_id": "polyline:20948",
"name": "Southeast Lambert Street",
"street": "Southeast Lambert Street",
"confidence": 0.8,
"match_type": "fallback",
"accuracy": "centroid",
"country": "United States",
"country_gid": "whosonfirst:country:85633793",
"country_a": "USA",
"region": "Oregon",
"region_gid": "whosonfirst:region:85688513",
"region_a": "OR",
"county": "Multnomah County",
"county_gid": "whosonfirst:county:102081631",
"locality": "Portland",
"locality_gid": "whosonfirst:locality:101715829",
"neighbourhood": "Brentwood-Darlington",
"neighbourhood_gid": "whosonfirst:neighbourhood:420781227",
"label": "Southeast Lambert Street, Portland, OR, USA"
},
"bbox": [
-122.609368,
45.467248,
-122.599598,
45.467255
]
},
{
"type": "Feature",
"geometry": {
"type": "Point",
"coordinates": [
-122.626516,
45.467079
]
},
"properties": {
"id": "polyline:20950",
"gid": "openstreetmap:street:polyline:20950",
"layer": "street",
"source": "openstreetmap",
"source_id": "polyline:20950",
"name": "Southeast Lambert Street",
"street": "Southeast Lambert Street",
"confidence": 0.8,
"match_type": "fallback",
"accuracy": "centroid",
"country": "United States",
"country_gid": "whosonfirst:country:85633793",
"country_a": "USA",
"region": "Oregon",
"region_gid": "whosonfirst:region:85688513",
"region_a": "OR",
"county": "Multnomah County",
"county_gid": "whosonfirst:county:102081631",
"locality": "Portland",
"locality_gid": "whosonfirst:locality:101715829",
"neighbourhood": "Eastmoreland",
"neighbourhood_gid": "whosonfirst:neighbourhood:85817069",
"label": "Southeast Lambert Street, Portland, OR, USA"
},
"bbox": [
-122.635486,
45.467069,
-122.617028,
45.468617
]
}
],
"bbox": [
-122.656975,
45.467069,
-122.599598,
45.468617
]
}
```

**Comment by fpurcell** [comment]
I'm not building anymore, so going to close this ticket.  (But need to reopen #25, since Interpolation has stopped working again.
